src/form.jsx

```diff
@@ -316,6 +327,10 @@ export default class Form extends React.Component {
     this.setState({ errorMessages: {} })
   }
 
+  clearForm () {
+    this.setState({doc: undefined, changes: undefined})
```

Comment:
for both doc and changes or only for changes?

I was thinking it might be good to use `doc` as used on the constructor, but I'm not sure if this is useful or not, as it is not being "cleaned".